dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/client/LibertyWebServiceClientInInterceptor.java

```diff
@@ -87,17 +78,32 @@ public void handleMessage(Message message) throws Fault {
                 
             }
 
-            
+            // Liberty change begin
```

Comment:
Since this is Liberty integration class, you technically don't need the `//Liberty Change Begin` comments like we put in override files, but its fine to keep it, but if other changes are necessary, you should probably drop them.